### PR TITLE
Fixed some tests

### PR DESCRIPTION
**Pull request by Saxton-Hale**

### CODE REVIEW

File: src/PDS.API.Tests/ChannelServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+namespace PDS.API.Tests;
+
+public class ChannelServiceTests
+{
+    [Test]
+    public async Task GetChannels()
+    {
+        var fixture = new InfrastructureFixture();
+        var stateService = fixture.ServiceProvider.GetService<IChannelService>();
```
Comment (robert-bialy) [inline src/PDS.API.Tests/ChannelServiceTests.cs]: Incorrect name, please change it to channelService.

File: src/PDS.API.Tests/ChannelServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+        var states = await stateService.GetChannels();
```
Comment (robert-bialy) [inline src/PDS.API.Tests/ChannelServiceTests.cs]: Incorrect name, please change to channel.

File: src/PDS.API.Tests/DetailServiceTest.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+public class DetailServiceTests
+{
+
+    [Test]
+    public async Task GetDetailsByPackageKey()
+    {
+
+        var fixture = new InfrastructureFixture();
+        var detailService = fixture.ServiceProvider.GetService<IDetailService>();
```
Comment (robert-bialy) [inline src/PDS.API.Tests/DetailServiceTest.cs]: Please change the name from IDetailService to IDetailsService. You can change variable names as well.

File: src/PDS.API.Tests/InfrastructureFixture.cs [hunk 4]
```diff
@@ -39,5 +39,29 @@ public InfrastructureFixture()
             consignment.BaseAddress = new Uri(apiUrl);
         });
         ServiceProvider = services.BuildServiceProvider();
```
Comment (robert-bialy) [inline src/PDS.API.Tests/InfrastructureFixture.cs:43]: ServiceProvider is being invoked in line 65. Please remove invocation here.

File: src/PDS.API/Models/Category.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Text.Json.Serialization;
+
+namespace PDS.API.Models;
+
+public class Category
+{
+
```
Comment (robert-bialy) [inline src/PDS.API/Models/Category.cs:7]: Remove newline.

File: src/PDS.API/Models/State.cs [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace PDS.API.Models;
+
+public class State
+{
+    [JsonPropertyName("value")]
+    public int Value { get; set; }
+    [JsonPropertyName("name")]
+    public string Name { get; set; }
+
```
Comment (robert-bialy) [inline src/PDS.API/Models/State.cs:11]: Remove newline.

File: src/PDS.API/Models/Statistics.cs [hunk 7]
```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json.Serialization;
+
+namespace PDS.API.Models;
+
+public class Statistics
+{
+    [JsonPropertyName("customerKey")]
+    public string CustomerKey { get; set; }
+
```
Comment (robert-bialy) [inline src/PDS.API/Models/Statistics.cs:9]: Remove newline.

File: src/PDS.API/Services/ChannelService.cs [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+using System.Threading.Channels;
+using PDS.API.Models;
+using Channel = PDS.API.Models.Channel;
+
+namespace PDS.API.Services;
+
+public interface IChannelService
+{
+    Task<Channel[]?> GetChannels();
+}
```
Comment (robert-bialy) [inline src/PDS.API/Services/ChannelService.cs:11]: Add newline between interface and class definition.

**Review by robert-bialy — CHANGES_REQUESTED** [review]

**Comment by robert-bialy** [comment]
@Saxton-Hale which of the tests are not working? Please give me a list of failing tests in the comment here.

**Comment by Saxton-Hale** [comment]
Tests which fail: ChannelServiceTests, all ConsignmentServiceTests and DetailServiceTests

**Pull request merged**